.github/workflows/prepare-for-minor-release.yml

```diff
@@ -137,14 +137,26 @@ jobs:
           .lando.yml
           .probo.yaml
           composer.json
-          
+
   update_previous_release_branches:
     needs: [prepare_for_new_minor_release, branch_name_prep]
     runs-on: ubuntu-latest
     name: Update `${{ matrix.previous_version }}` release branch.
```

Comment:
`name: Update `${{ matrix.previous_version }}` release branch.` is probably not what we want to see in the PR for main.